ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1167,20 +1229,24 @@ export const WorkspaceEdit = fp.flow(withCurrentWorkspace(), withCdrVersions(),
               </div>
             </OverlayPanel>
             <FlexRow>
-              <Dropdown style={{width: '14rem'}}
+              {this.state.fetchBillingAccountLoading ? <SpinnerOverlay overrideStylesOverlay={styles.spinner}/> :
+                  <Dropdown data-test-id = 'billing-dropdown'
+                      style={{width: '20rem', overlayVisible: 'true'}}
```

Comment:
was overlayvisible from experimentation, or if not - why is it needed now?

Reply:
that might not needed.